.github/workflows/mindsdb.yml

```diff
@@ -62,6 +62,7 @@ jobs:
           pip install git+https://github.com/mindsdb/lightwood.git@staging --upgrade --no-cache-dir
           pip uninstall pycodestyle -y && pip install pycodestyle==2.9.1  # temp fix while exist conflict in last version with flake8
           pip install gluonts==0.11.4  # temp fix until next lightwood release (>23.1.2.1)
+          pip install statsforecast==1.4.0
```

Comment:
We're internally discussing how to best approach this. For now it's okay, but this will probably change.

Reply:
Gotcha - agree it's a bit dirty. Happy to contribute to these discussions if helpful - had similar experience with dependency hell in CI (we set up a separate ci_reqs.txt file in the end)